test/cypress/e2e/search.spec.js

```diff
@@ -3,11 +3,10 @@ import networks from '../../constants/networks';
 import ss from '../../constants/selectors';
 import urls from '../../constants/urls';
 
-const getSearchesObjFromLS = () => JSON.parse(localStorage.getItem('searches'));
-
 describe('Search', () => {
   const testnetTransaction = '755251579479131174';
   const mainnetTransaction = '881002485778658401';
+  const testnetTransactionId = '6676752260506338126';
```

Comment:
How this transaction is different from the one 2 lines above? Do you need a second one?

Reply:
fixed